Taskfile.yml

```diff
@@ -29,39 +33,132 @@ tasks:
       - mkdir -p {{.LOCATION}}
       - mv {{.BINARY}} {{.LOCATION}}/{{.BINARY}}
 
+  ci:
+    desc: Workflow to run in CI
+    cmds:
+      - task: go:setup
+      - task: go:has-latest-opslevel-go
+      - task: lint
+      - task: test
+
+  clean:
+    desc: Clean up terraform files from "{{.WORKSPACE_DIR}}"
+    dir: "{{.WORKSPACE_DIR}}"
+    prompt: Remove '*.tfstate .terraform.lock.hcl ./terraform.* .terraform/**' from '{{.WORKSPACE_DIR}}' directory?
+    cmds:
+      - cmd: rm -rf *.tfstate .terraform.lock.hcl ./terraform.* .terraform/
+        ignore_error: true
+
+  destroy:
+    desc: Run "terraform destroy" in "{{.WORKSPACE_DIR}}"
+    cmds:
+      - task: terraform-command
+        vars: { TF_COMMAND: "destroy", TF_CMD_DIR: "{{.WORKSPACE_DIR}}" }
+
+  fix:
+    desc: Fix formatting and linting
+    cmds:
+      - task: go:fix
+      - task: format-fix
+
+  init:
+    desc: Initialize terraform workspace
+    dir: "{{.WORKSPACE_DIR}}"
+    cmds:
+      - task: install-terraform
+      - './make_backend_tf.sh'
+      - touch main.tf
+      - task: terraform-command
+        vars: { TF_COMMAND: "init -upgrade", TF_CMD_DIR: "{{.WORKSPACE_DIR}}" }
+
   install-changie:
     desc: Install changie
     aliases: [get-changie]
     cmds:
       - cmd: echo "Installing changie..."
         silent: true
-      - go install github.com/miniscruff/changie@latest
-      - cmd: echo -e "\nSuccess! 'changie' installed."
+      - which changie > /dev/null || go install github.com/miniscruff/changie@latest
+      - cmd: echo -e "Success! 'changie' installed."
         silent: true
 
   lint:
     desc: Check formatting and linting
     cmds:
       - task: go:lint
-      - task: terraform:lint
+      - task: validate
+      - task: format-check
```

Comment:
`format-check` should not be part of linting since the data in workspace/ doesn't matter - it's only used for tophatting.

Reply:
This runs `terraform fmt -recursive -check` from the project root directory, which includes all of the `examples/**/*.tf` files and any future `*.tf` files that we add with the framework upgrade. Terraform `1.7` just released with improvements to `test { }` blocks and this will be needed for those too

Reply:
I was not aware of that - good idea